jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -404,4 +423,23 @@ public void onResize() {
             listTable.onResize();
         }
     }
-}
\ No newline at end of file
+
+    @Override
+    public ColumnSortList reloadColumnSortList() {
+        ColumnSortList columnSortList = getListGrid().getColumnSortList();
+        GridPreferencesStore gridPreferencesStore = listTable.getGridPreferencesStore();
+        GridSortedColumnPreference gridSortedColumnPreference = gridPreferencesStore.getGridSortedColumnPreference();
+        if (gridPreferencesStore != null && gridSortedColumnPreference != null && columnSortList.size() <= 1) {
```

Comment:
I believe the current _columnSortList_ is irrelevant for the if statement.

Reply:
but you only execute this creating the table right? that why my question... if just reloading the settings, you should override or ignore what is previously selected.

Reply:
The method will call by getData() of AbstractMultiGridPresenter, and the method will be call many event , like init and Column sorted event, I find if size of columnSortList more than 2 , it mean, here catch ColumnSortedEvent, so I ignore all size more than 2's case.